### ISSUE TITLE
what controls back button and fab-home? what are header fab?

### ISSUE THREAD

**Issue by abioneperhobby** [zachfitz/Ionic-Material#189]
Hi

i have some difficults to understand some things : 

1) what controls back button? i make a copy of activity. if i reach activity by menu.html , as for example

`
<a ui-sref="app.activity" class="tab-item"><i class="icon ion-arrow-graph-up-right"></i> Activity</a>
`
the reached view has back button,
![image](https://cloud.githubusercontent.com/assets/2149815/14768659/ac86e0b6-0a48-11e6-9e8f-ac68dce9308f.png)
but if i reach the same view by right menu with
`
<ion-item nav-clear menu-close ui-sref="app.activity">Activity</ion-item>
`
there is no back button. 

what does controls the back button ?

2)

i modified app.js in order to launch a view called matches on a click. 

`
template: '<button id="fab-home" ui-sref="app.matches" class="button button-fab button-fab-bottom-right button-energized-900"><i class="icon ion-plus"></i></button>',
`

2a) matches view doesn't have back button (similar to problem1). why?
2b) matches view contains again the same "fab-home" button. why?
![image](https://cloud.githubusercontent.com/assets/2149815/14768661/b99953a6-0a48-11e6-862c-9a816860d501.png)

3)

controllers.js contains $scope.setHeaderFab function. what is the intent of this function? what is a "fab"? 

MANY THANKS for any help.

**Comment by abioneperhobby** [comment]
please, someone can help me ? who set the hide class?
![image](https://cloud.githubusercontent.com/assets/2149815/14923883/1dff2424-0e40-11e6-9d87-9e451a8920b2.png)

**Comment by abioneperhobby** [comment]
problem was related to nav history.

i don't know why but entering in a state by the right menu, there is no "history back".

so i removed ui-sref=XXX and i added a function that calls $state.go(XXX) so history has a back status and back button can be used.

**Comment by neslinesli93** [comment]
I had a very similar problem: instead of the back button, it happened to me with the main menu botton.
After some state change, all of a sudden it disappeared because the class hide was randomly applied to it.
Long story short, I had to revisit my application navbar, since probably it was a problem with the ionic overrides this library makes.
I think I solved it by removing all the "hide-back-button" classes, and by adding back buttons and menu buttons manually in every view
